app/src/main/java/com/example/habitsmasher/User.java

```diff
@@ -10,7 +11,7 @@
  *
  * Currently, profile picture is not used
  */
-public class User {
+public class User implements Serializable {
```

Comment:
Implement Serializable so we can pass the whole User object to different Fragments, not just the username